backend/src/test/java/com/huellapositiva/api/EmailAddressConfirmationControllerShould.java

```diff
@@ -60,10 +64,12 @@ void confirm_email_with_non_existing_hash_should_return_404() throws Exception {
     void resend_email_should_return_204() throws Exception {
         //GIVEN
         testData.createCredential(DEFAULT_EMAIL, UUID.randomUUID(), DEFAULT_PASSWORD, VOLUNTEER_NOT_CONFIRMED);
+        MockHttpServletResponse loginResponse = loginRequest(mvc, new CredentialsVolunteerRequestDto(DEFAULT_EMAIL, DEFAULT_PASSWORD));
 
         // WHEN + THEN
         mvc.perform(post(baseUri + "/resend-email-confirmation")
                 .with(user(withMockUser(DEFAULT_EMAIL, VOLUNTEER_NOT_CONFIRMED)))
+                .with(csrf())
```

Comment:
In tests where we do not want to test CSRF explicitly it is better to simulate it through Spring test utils to have a cleaner test that focuses only in the task we want to achieve. The same applies for logged in users.